client/consensus/common/src/lib.rs

```diff
@@ -67,6 +75,15 @@ impl<B: BlockT> ParachainConsensus<B> for Box<dyn ParachainConsensus<B> + Send +
 	) -> Option<ParachainCandidate<B>> {
 		(*self).produce_candidate(parent, relay_parent, validation_data).await
 	}
+
+	async fn is_collating(
+		&mut self,
+		parent: &B::Header,
+		relay_parent: PHash,
+		validation_data: &PersistedValidationData,
+	) -> Option<CollationForecast> {
+		(*self).is_collating(parent, relay_parent, validation_data).await
+	}
```

Comment:
@bkchr should we preconnect for relay-chain consensus? Seems like we always want to do this if the core is free, right?

Reply:
You added the comment to the wrong position, but yeah for the "relay-chain consensus" you always want to connect.